releasenotes/notes/prf-stack-v2-adaptive-sampling-38cd6af0882fc36b.yaml

```diff
@@ -0,0 +1,30 @@
+---
+#instructions:
```

Comment:
Mind if we cut out all of the comments here? I'm not sure our script is smart enough to deal with those... although if you've done before and it was fine feel free to ignore.

Reply:
ah whoops I forgot!